packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1029,7 +1029,9 @@ function commitUnmount(
       if (enableDeprecatedFlareAPI) {
         unmountDeprecatedResponderListeners(current);
       }
-      beforeRemoveInstance(current.stateNode);
+      if (current.ref !== null) {
```

Comment:
Can we put this in the feature flag? The benefit is that custom renderer users won't have to implement a noop method.

Reply:
I can wrap it with a feature flag, but I still need to export it from each of the renderers or Flow throws an error.